hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/router/TestRouterSafemode.java

```diff
@@ -128,7 +129,6 @@ public void testRouterExitSafemode()
 
     assertTrue(router.getSafemodeService().isInSafeMode());
     verifyRouter(RouterServiceState.SAFEMODE);
-
```

Comment:
avoid this change

Reply:
Thanks @slfan1989 help me reivew it, i wll update it later.